tests/chainer_tests/test_function_node.py

```diff
@@ -960,13 +965,12 @@ def check_grad(self):
             self._print_variables('gxs (expected)', expected)
             raise
 
-    def test_grad_cpu(self):
-        self.check_grad()
-
-    @attr.gpu
-    def test_grad_gpu(self):
-        self.use_gpu()
-        self.check_grad()
+    def test_grad(self, backend_config):
+        if self.loss_scale and backend_config.xp is chainerx:
+            pytest.skip('chainerx.grad does not support loss_scale')
+        self.use_device(backend_config.device)
+        with chainer.using_device(backend_config.device):
+            self.check_grad()
```

Comment:
`chainer.gad` should not require `using_device`.
Let's apply `using_device` to where it is necessary, not to the whole check function.

Reply:
There was a bug in function_node that appeared when adding multigpu tests.